sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -89,6 +90,32 @@ export class ApplicationAESTController extends BaseController {
     );
   }
 
+  /**
+   * API to fetch application header information by applicationId.
+   * This API will be used by ministry users.
+   * @param applicationId
+   * @returns Application header information
+   */
+  @Get(":applicationId/header")
+  @ApiNotFoundResponse({ description: "Application not found." })
+  async getApplicationHeaderInformation(
+    @Param("applicationId", ParseIntPipe) applicationId: number,
+  ): Promise<ApplicationHeaderAPIOutDTO> {
+    const application = await this.applicationService.getApplicationById(
+      applicationId,
+      { loadDynamicData: true },
+    );
+    if (!application) {
+      throw new NotFoundException(
+        `Application id ${applicationId} was not found.`,
+      );
+    }
+
+    return this.applicationControllerService.transformToApplicationHeaderForAESTDTO(
+      application,
+    );
+  }
```

Comment:
Please remove this code and take the latest from main. 

Code from PR https://github.com/bcgov/SIMS/pull/3730 has modified the existing API with option to use `loadDynamicData` false and built the header component.

Reply:
Removed.